proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -59,9 +84,11 @@ message MsgCreateValidator {
   option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
 
-  Description     description         = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
-  CommissionRates commission          = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
-  string          min_self_delegation = 3 [
+  Description     description = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+  CommissionRates commission  = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+  // Deprecated: This field has been deprecated with LSM in favor of the validator bond
+  string min_self_delegation = 3 [
+    deprecated             = true,
```

Comment:
Add `deprecated = true,` and `(gogoproto.nullable) = false` to `min_self_delegation` in `message MsgEditValidator` below.